platform-dependent-components/problem-solver/cxx/identifiers-module/agent/translate_main_system_idtfs_from_sc_to_file_agent.cpp

```diff
@@ -151,7 +150,7 @@ bool TranslateMainSystemIdtfsFromScToFileAgent::WriteInFile(std::string const &
 {
   try
   {
-    std::ofstream file(IDENTIFIERS_MODULE_PATH "identifiers.txt");
```

Comment:
why is IDENTIFIERS_MODULE_PATH deleted?

Reply:
This variable was set at compile time, which prevents this agent from being used on other devices, as this variable will store the absolute path